### PR TITLE
REF: Move general tools under wallet details to "tools" menu

### PR DESCRIPTION
**Pull request by ncoelho**
A little cleaning on the details view by moving the generalist tools that do not depend on a wallet to a upper level menu under the settings view.

Screenshot 2021-03-30 at 20 12 12 | Screenshot 2021-03-30 at 20 12 22

### CODE REVIEW

**Comment by marcosrdz** [comment]
Lint failed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tua7map5zwj1xvxja1zdzg0twg

[download apk](http://filestorage.bluewallet.io:1488/toolsmenu-1617136388.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/7kmab10jfm0yygfgtczczqq4b0

[download apk](http://filestorage.bluewallet.io:1488/toolsmenu-1617205200.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6d6x229z1xcd7912k6r37zt1xw

[download apk](http://filestorage.bluewallet.io:1488/toolsmenu-1617209561.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wj17fxty5wyg9knmuxrkyw9wp4

[download apk](http://filestorage.bluewallet.io:1488/toolsmenu-1617272760.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/fhzxwwgy984b1exrgg1q2grumm

[download apk](http://filestorage.bluewallet.io:1488/toolsmenu-1617285913.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/uxu0er58zebxkc36vp623rz4dw

[download apk](http://filestorage.bluewallet.io:1488/toolsmenu-1617357840.apk)

**Comment by Overtorment** [comment]
tools->privacy is a weird place to put privacy settings.

File: screen/settings/NetworkSettings.js [hunk 1]
```diff
@@ -17,16 +17,19 @@ const NetworkSettings = () => {
...
-
   return (
     <SafeBlueArea>
       <ScrollView>
         <BlueListItem title={loc.settings.network_electrum} onPress={navigateToElectrumSettings} testID="ElectrumSettings" chevron />
         <BlueListItem title={loc.settings.lightning_settings} onPress={navigateToLightningSettings} testID="LightningSettings" chevron />
-        <BlueListItem title={loc.settings.network_broadcast} onPress={navigateToBroadcast} testID="Broadcast" chevron />
+        {Notifications.isNotificationsCapable && (
+        <BlueListItem
+          title={loc.settings.notifications}
```
Comment (Overtorment) [inline screen/settings/NetworkSettings.js:27]: needs lint

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bwy40dgxfz6h5jbb5u712n2f5w

[download apk](http://filestorage.bluewallet.io:1488/toolsmenu-1617363642.apk)

**Comment by marcosrdz** [comment]
Privacy options in Settings are toggles to disable or enable something. They aren't tools for execution of an action. Seems like its in the wrong place

**Comment by ncoelho** [comment]
I was thinking of moving them under the general options. They are also to small to have as a full section and we already have too many options, we can't really add more.

**Comment by marcosrdz** [comment]
That seems like a more adequate solution.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zdmherj4k9r0e7kbumb3499e58

[download apk](http://filestorage.bluewallet.io:1488/toolsmenu-1617707058.apk)

**Review by limpbrains — APPROVED** [review]

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**